doc/changelog/11-infrastructure-and-dependencies/15483-bump-zarith-doc.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  Minimal supported zarith version is now 1.12
```

Comment:
```suggestion
  Minimum supported zarith version is now 1.12
```

Not the right word.  Could say "Lowest", too.

Reply:
Could you elaborate why you don't think it is right? It sounds fine to my ear.

Reply:
(Both words also have a mathematical use which is correct in this context.)

Reply:
Despite the debate, I have committed the suggestion.

Reply:
"Minimum", meaning "the least or smallest amount or quantity possible, attainable, or **required**" (Oxford) is precisely what's needed here.  "Minimal", meaning "of a minimum amount, quantity, or degree; **negligible**" (Oxford) is a bit off.  Certainly "negligible" is not what's intended.  I think mathematical use is not so relevant here, this section is not at all mathematical.  IMHO.